speech/google/cloud/speech/streaming/response.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Representation of a GAPIC Speech API response."""
+
+from google.cloud.speech.streaming.result import StreamingSpeechResult
+
+
+class StreamingSpeechResponse(object):
+    """Representation of a Speech API protobuf streaming response.
+
+    :type error: :class:`google.grpc.Status`
+    :param error: Instance of ``Status``
+
+    :type endpointer_type: :class:`~EndpointerType`
+    :param endpointer_type: Enum of endpointer event.
+
+    :type results: list of
+        :class:`google.cloud.speech.v1beta1.StreamingRecognitionResult`
+    :param results: List of protobuf ``StreamingRecognitionResult``.
+
+    :type result_index: int
+    :param result_index: Index for specific result set. Used for updating with
+                         ``interim_results``.
+    """
+    def __init__(self, error, endpointer_type, results, result_index):
+        self._error = error
+        self._endpointer_type = endpointer_type  # Should be enum.
```

Comment:
This comment is inappropriate here? Do you have a condition like to enforce: then check the type and raise an error if so. Do you have a type you'd like to document: then put in the docstring.